src/goto-programs/system_library_symbols.cpp

```diff
@@ -246,6 +258,20 @@ void system_library_symbolst::add_to_system_library(
   }
 }
 
+/// Helper function to call `is_symbol_internal_symbol` on a nameless
+/// fake symbol with the given type, to determine whether the type alone
+/// is sufficient to classify a symbol of that type as internal.
+/// \param type: the type to check
+/// \return True if the symbol is an internal symbol. If specific system headers
```

Comment:
`/// \return True if the `**`type`**` is an internal `**`type`**`. If specific system headers`